test/actions/katello/repository_test.rb

```diff
@@ -181,6 +181,45 @@ class DiscoverTest < TestBase
     end
   end
 
+  class ContentUpdateTest < TestBase
+    let(:action_class) { ::Actions::Katello::Repository::ContentUpdate }
+    let(:repository) { katello_repositories(:fedora_17_unpublished) }
+    let(:candlepin_action_class) { ::Actions::Candlepin::Product::ContentUpdate }
+    let(:candlepin_enablement_class) { ::Actions::Candlepin::Product::ContentUpdateEnablement }
+    let(:action_planned_yes) { create_and_plan_action action_class, repository, update_auto_enabled: true }
+    let(:action_planned_no) { create_and_plan_action action_class, repository }
+
+    def setup
+      content = FactoryBot.create(:katello_content, cp_content_id: repository.content_id, organization_id: repository.product.organization_id)
+      Katello::ProductContent.create!(:content_id => content.id, :product_id => repository.product_id)
+      super
+    end
+
+    it 'plans' do
+      action = create_action action_class
+      plan_action action, repository, update_auto_enabled: true
+      assert_action_planed action, candlepin_action_class
```

Comment:
```suggestion
      assert_action_planned action, candlepin_action_class
```
How does it pass like this?  :confused:

Reply:
oh it is assert_action_planed was in the original dynflow https://github.com/Dynflow/dynflow/blob/master/lib/dynflow/testing/assertions.rb#L44 :) . Both work since they are aliases. If you looked the asserts in this file you will see assert_action_planed everywhere